api/pending_test_runs.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// apiPendingTestRunsHandler is responsible for emitting JSON for
+// all the pending test runs.
+func apiPendingTestRunsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := shared.NewAppEngineContext(r)
+	store := shared.NewAppEngineDatastore(ctx, true)
```

Comment:
```suggestion
	store := shared.NewAppEngineDatastore(ctx, false)
```
We probably don't want to cache this because pending runs are mutable (and will change frequently in their lifecycle).

Reply:
I think the cache only affects `TestRun` entities in any case, but sure.